_dashboards/index.md

```diff
@@ -11,15 +11,54 @@ redirect_from:
 
-## Get started with OpenSearch Dashboards
+##### Example: Data visualizations in OpenSearch Dashboards  
 
-1. After starting OpenSearch Dashboards, you can access it at port 5601. For example, http://localhost:5601.
-1. Log in with the default username `admin` and password `admin`.
-1. Choose **Try our sample data** and add the sample flight data.
-1. Choose **Discover** and search for a few flights.
-1. Choose **Dashboard**, **[Flights] Global Flight Dashboard**, and wait for the dashboard to load.
+<img src="{{site.url}}{{site.baseurl}}/images/dashboard-flight.png" alt="User interface showing several data visualizations">
+
+# Use cases for real-time search and log analytics at scale 
+
+ OpenSearch is powered by the Apache Lucene search library, and it supports a number of search and analytics capabilities, such as k-nearest neighbors (k-NN) search, SQL, anomaly detection, trace analytics, full-text search, and more.  
+
+##### Example: Data visualization use cases   
+
+<table style="table-layout: fixed; width: 100%;">
+<tbody>
+<tr>
+<td style="text-align: center; vertical-align: top; font-weight: bold; width: 100%;">Visualize log and trace data with interactive log analytics.<img src="{{site.url}}{{site.baseurl}}/images/visualize-log-data.png" alt="Log analytics" /></td>
+<td style="text-align: center; vertical-align: top; font-weight: bold; width: 100%;">Detect and mitigate issues faster with anomaly detection.<img src="{{site.url}}{{site.baseurl}}/images/anomaly-detection.png" alt="Anomaly detection" /></td>
+</tr>
+<tr>
+<td style="text-align: center; vertical-align: top; font-weight: bold; width: 100%;">Diagnose performance issues and reduce application downtime using the observability interface and log monitoring features.<img src="{{site.url}}{{site.baseurl}}/images/observability.png" alt="Observability interface and log monitoring features" /></td>
+<td style="text-align: center; vertical-align: top; font-weight: bold; width: 100%;">Unlock real-time search, monitoring, and analysis of business and operational data.<img src="{{site.url}}{{site.baseurl}}/images/analyzing-data-logs.png" alt="Real-time search, monitoring, and analysis" /></td>
+</tr>
+</tbody>
+</table> 
+
+# Explore the OpenSearch Dashboards playground
+
+You can interact with the three demonstration datasets in the OpenSearch Dashboards playground by following the steps below:
+
+1. Go to the [OpenSearch Dashboards playground](https://playground.opensearch.org/app/home).
+2. Choose **OpenSearch Dashboards > Dashboard**. 
+3. Explore the datasets **[Flights] Global Flight Dashboard**, **[Logs] Web Traffic**, and **[eCommerce] Revenue Dashboard**.
+4. Interact with the data on a dashboard. Choose **Add filter** at top left, and then choose a field that you want to filter, along with an operator and a value.  
```

Comment:
Can we be more specific than "at top left"?

Reply:
Deleted since it's the only Add filter on the page.

Reply:
Revised to create a step 5, breaking up the multiple steps in step 4.